nixos/modules/services/x11/display-managers/startx.nix

```diff
@@ -39,6 +39,14 @@ in
       displayManager.lightdm.enable = lib.mkForce false;
     };
     systemd.services.display-manager.enable = false;
+
+    # Let Xorg use its default log location ($XDG_DATA_HOME/xorg/)
+    services.xserver.logFile = mkDefault null;
```

Comment:
Why do we only do this when using a display manager?

Reply:
I found this commit which originally set the `/dev/null` default: d84741a4bfb6. Is there a reason to change the default here?

Reply:
The Startx "displayManager" is not a systemd service and therefore doesn't log to journal when the logfile is set to /dev/null IIRC.

Startx logging to your home dir is what users would expect it to do.

Reply:
That makes a lot of sense :) Please add that reasoning in a comment as well.